unittests/data/unnamed_classes.hpp

```diff
@@ -25,7 +25,7 @@ struct S1{
 			unsigned int raw;
 		} flags;
 
-		union FileAttribs{
+		union /*FileAttribs*/{
```

Comment:
Makes sense. I'll just remove the commented out part then, so nobody wonders why it is there. Thanks